avaframe/ana3AIMEC/dfa2Aimec.py

```diff
@@ -75,7 +73,7 @@ def extractMBInfo(avaDir):
         flagStop = 0
 
         # Read log file
-        with open(os.path.join(os.getcwd(), avaDir, 'Outputs', 'com1DFA', 'start%s.log' % (relName)), 'r') as file:
+        with open(os.path.join(os.getcwd(), avaDir, 'Outputs', 'com1DFA', 'start%s.log' % (simName)), 'r') as file:
```

Comment:
Just a maybe (for readability)
construct the file name in a separate line...

Reply:
Done